src/sdlwindow.cpp

```diff
@@ -879,6 +882,10 @@ namespace gamescope
 						if ( g_bGrabbed )
 							szTitle += " (grabbed)";
 						SDL_SetWindowTitle( m_Connector.GetSDLWindow(), szTitle.c_str() );
+						SDL_SetHint(
+							SDL_HINT_SCREENSAVER_INHIBIT_ACTIVITY_NAME,
+							(std::string("Title: ") + szTitle).c_str() );
```

Comment:
Can you pull the std::string construction out above and then take the .c_str().

Maybe this isn't a problem in the more, but I've definitely had issues with lifetimes and stuff doing exactly this.

Reply:
done!